### PR TITLE
feat(api-1652): update postman collection

### PR DESCRIPTION
**Pull request by tfehringer**
API-1652, API-1653, API-1654

### CODE REVIEW

**Review by dFayet — APPROVED** [review]

File: content/files/Akeneo PIM API.postman_collection.json [hunk 1]
```diff
@@ -2869,6 +2869,158 @@
                 "rest",
                 "v1",
                 "products"
+              ],
```
[the remark this answers is not in the document]
Comment (tfehringer) [reply content/files/Akeneo PIM API.postman_collection.json:2872]: I didn't find any of those query parameters in the filter's documentation. Tickets are only about updating filters not query params right ? Should i add examples for all query params as you listed ?

[the remark this answers is not in the document]
Comment (benjaminleneve) [reply content/files/Akeneo PIM API.postman_collection.json:2872]: Yes, it is possible that it is out of the scope of the ticket. It's just that I think we can take advantage of it to have all the query parameters in the collection. What do you think @meganelepalud-akeneo  ?

[the remark this answers is not in the document]
Comment (meganelepalud-akeneo) [reply content/files/Akeneo PIM API.postman_collection.json:2872]: Good initiative to add the available options 👍

**Review by dFayet — APPROVED** [review]

File: content/files/Akeneo PIM API.postman_collection.json [hunk 2]
```diff
@@ -2869,6 +2869,206 @@
                 "rest",
                 "v1",
                 "products"
```
Comment (benjaminleneve) [inline content/files/Akeneo PIM API.postman_collection.json:2871]: We should also add the following query parameters on GET /api/rest/v1/products/{code} :
- with_attribute_options
- with_quality_scores
- with_completenesses

File: content/files/Akeneo PIM API.postman_collection.json [hunk 3]
```diff
@@ -3133,6 +3333,86 @@
                 "rest",
                 "v1",
                 "product-models"
```
Comment (benjaminleneve) [inline content/files/Akeneo PIM API.postman_collection.json:3335]: We should also add the following query parameters on GET /api/rest/v1/product-models/{code}:
- with_quality_scores

**Review by benjaminleneve — APPROVED** [review]

**Review by meganelepalud-akeneo — APPROVED** [review]

**Pull request merged**